docs/ARCHITECTURE.md

```diff
@@ -44,9 +43,9 @@ graph LR
 
 ## Concepts
 
-The core of the PDC is its API.  All third party integrations and user interfaces are powered by that API.
```

Comment:
oh no it removes double spaces :( cc @reefdog

Reply:
I'm very happy for your loss.